components/library/SDL2_mixer/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2016 Jim Klimov
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		SDL2_mixer
+COMPONENT_VERSION=	2.0.1
+COMPONENT_PROJECT_URL=	http://www.libsdl.org/projects/SDL_mixer/
+COMPONENT_SUMMARY=	SDL_mixer is a sample multi-channel audio mixer library
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH=	\
+    sha256:5a24f62a610249d744cbd8d28ee399d8905db7222bf3bdbc8a8b4a76e597695f
+COMPONENT_ARCHIVE_URL=	http://www.libsdl.org/projects/SDL_mixer/release/$(COMPONENT_ARCHIVE)
+COMPONENT_FMRI=	library/audio/sdl2-mixer
+COMPONENT_LICENSE =	ZLIB
+COMPONENT_LICENSE_FILE =	COPYING.txt
+COMPONENT_CLASSIFICATION =	System/Multimedia Libraries
+
+include $(WS_TOP)/make-rules/prep.mk
+include $(WS_TOP)/make-rules/configure.mk
+include $(WS_TOP)/make-rules/ips.mk
+
+PATH=/usr/gnu/bin:/usr/bin
+
+COMPONENT_PREP_ACTION = ( cd $(@D) && ./autogen.sh )
+
+CONFIGURE_OPTIONS +=	--sysconfdir=/etc
+CONFIGURE_OPTIONS +=	--libexecdir=$(CONFIGURE_LIBDIR.$(BITS))
+CONFIGURE_OPTIONS +=	--enable-music-cmd
+CONFIGURE_OPTIONS +=	--enable-music-wave
+CONFIGURE_OPTIONS +=	--enable-music-midi
+CONFIGURE_OPTIONS +=	--enable-music-mod-mikmod --enable-music-mod-mikmod-shared
+CONFIGURE_OPTIONS +=	--enable-music-ogg-shared
+CONFIGURE_OPTIONS +=	--enable-music-flac-shared
+
+### Note: currently we do not serve libmad, and MPEG support is reserved for
+### encumbered components
+#CONFIGURE_OPTIONS +=	--enable-music-mp3-smpeg --enable-music-mp3-smpeg-shared
```

Comment:
fixed better in a recent commit

2016-04-28 0:54 GMT+02:00 Alexander Pyhalov <notifications@github.com>:

> In components/library/SDL2_mixer/Makefile
> <https://github.com/OpenIndiana/oi-userland/pull/1918#discussion_r61350671>
> :
>
> > +PATH=/usr/gnu/bin:/usr/bin
> > +
> > +COMPONENT_PREP_ACTION = ( cd $(@D) && ./autogen.sh )
> > +
> > +CONFIGURE_OPTIONS +=	--sysconfdir=/etc
> > +CONFIGURE_OPTIONS +=	--libexecdir=$(CONFIGURE_LIBDIR.$(BITS))
> > +CONFIGURE_OPTIONS +=	--enable-music-cmd
> > +CONFIGURE_OPTIONS +=	--enable-music-wave
> > +CONFIGURE_OPTIONS +=	--enable-music-midi
> > +CONFIGURE_OPTIONS +=	--enable-music-mod-mikmod --enable-music-mod-mikmod-shared
> > +CONFIGURE_OPTIONS +=	--enable-music-ogg-shared
> > +CONFIGURE_OPTIONS +=	--enable-music-flac-shared
> > +
> > +### Note: currently we do not serve libmad, and MPEG support is reserved for
> > +### encumbered components
> > +#CONFIGURE_OPTIONS +=	--enable-music-mp3-smpeg --enable-music-mp3-smpeg-shared
>
> Perhaps, you should explicitly disable them, as they are enabled by
> default if found
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenIndiana/oi-userland/pull/1918/files/81560a25413a8462344cac05e3e1d10694ea1af3#r61350671>
>